web/profiles/joinup/joinup.info.yml

```diff
@@ -39,7 +40,6 @@ dependencies:
   - rdf_entity
   - rdf_export
   - rdf_taxonomy
-  - solution
   - state_machine
```

Comment:
As I have seen, Drupal does not respect the dependencies during the profile installation so the modules must be put there appropriately.
So for example, before this ticket, we were installing the collection, and then the joinup news. But not that the solution is an og, news's field og_group_ref, got another dependency, which is the solution entity. This immediately broke my installation due to unmet dependencies.